src/main.rs

```diff
@@ -32,9 +35,23 @@ pub struct Cli {
 }
 
 fn main() -> Result<(), Error> {
+    setup_panic!();
+
     let args = Cli::from_args();
 
-    setup_panic!();
+    env_logger::Builder::new()
+        .filter(
+            None,
+            match args.verbosity {
+                0 => LogLevel::Error,
+                1 => LogLevel::Warn,
+                2 => LogLevel::Info,
+                3 => LogLevel::Debug,
+                _ => LogLevel::Trace,
+            }.to_level_filter(),
+        )
+        .try_init()?;
```

Comment:
We have nothing using `log` macros. What value is this providing?

Reply:
all the warns in cargo_ebuild::build are log. Log will be used more in future.